wiki/Reporting_Bad_Behaviour/Abuse/en.md

```diff
@@ -0,0 +1,66 @@
+# Reporting Abuse
+
+## I don't know if I should report this or not
+
+Yes, you should!
+
+If you're uncomfortable enough that you're reading this page, please contact us at once at [abuse@ppy.sh](mailto:abuse@ppy.sh) with as much detail about your problem as you can.
+
+We have zero tolerance towards any sort of unwanted sexual attention or advances in this community.
+
+The abuse email address is only seen by these two users:
+
+- [peppy](https://osu.ppy.sh/users/2) (the creator of the game)
+- [Ephemeral](https://osu.ppy.sh/users/102335) (the community manager)
+
+Nothing you send to [abuse@ppy.sh](mailto:abuse@ppy.sh) will ever be shared with anybody else other than those two people.
+
+## Someone's made inappropriate sexual advances or remarks towards me!
+
+Scroll through the sub-headings below to find out what to do, depending on where this all happened.
+
+### In the in-game chat
+
+Immediately block the user by right-clicking their username and selecting ``Ignore User``.
+
+You should also report them immediately by clicking the ``Report User`` button and filling in the prompts.
+
+If you feel the remarks/advances were highly inappropriate or you are an underage user, please send an email to [abuse@ppy.sh](mailto:abuse@ppy.sh) containing as much information as you can.
+
+### In a public post on the osu! forums
+
+Report the post to any member of our [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) as soon as possible.
+
+To do so, click the username of any GMT member in the list on that page. This will take you to their profile. If you are unsure of who to click, just use [this link instead](https://osu.ppy.sh/users/102335).
```

Comment:
perhaps you should note that the link belongs to your (Ephemeral's) userpage somewhere?

Reply:
i should probably replace this with a sendto chat link instead actually

Reply:
This method of reporting is pretty shoddy

Reply:
is there a better method for forum posts?

Reply:
another method for reporting forum posts is using the report button found on the user page and attaching the link there. that can also be mentioned here aside from reporting to a GMT through PMs, which some users prefer to do so since there isn't a report button specifically for forum posts/threads

Reply:
There isn't, just saying we probably want to get that fixed at some point.